examples/import_handmade.py

```diff
@@ -46,6 +51,7 @@ def import_events(client, file):
         entity_id=data[0],
         target_entity_type="item",  # type of item in this action
         target_entity_id=data[2],
+        event_time=timestamp
       )
       print "Event: " + data[1] + " entity_id: " + data[0] + " target_entity_id: " + data[2]
     elif (data[1] == "$set"):  # must be a set event
```

Comment:
This import script is used for the other integration test, does that still work? We may want to have an import per test so they can be altered without worrying about affecting the other tests.